cli/features/tutorial.feature

```diff
@@ -44,35 +44,37 @@ Feature: Following the tutorial
         public:
         private:
       """
+    And my workspace contains the empty directory "todo-app/.exosphere"
 
 
   Scenario: adding the html service
     Given I cd into "todo-app"
-    When starting "exo add service --template-name=htmlserver-express-es6" in this application's directory
+    And my application contains the template folder ".exosphere/exoservice-es6-mongodb"
+    And my application contains the template folder ".exosphere/htmlserver-express-es6"
+    When starting "exo add" in this application's directory
     And entering into the wizard:
-      | FIELD                         | INPUT                           |
-      | Role of the service to create | html-server                     |
-      | Type of the service to create | html-server                     |
-      | Description                   | serves HTML UI for the test app |
-      | Author                        | test-author                     |
-      | Name of the data model        |                                 |
-      | Protection level              | public                          |
+      | FIELD                         | INPUT                            |
+      | template                      | 2                                |
+      | serviceRole                   | html-server                      |
+      | appName                       | test-app                         |
+      | serviceType                   | html-server                      |
+      | description                   | serves HTML UI for the test app  |
+      | author                        | test-author                      |
+      | Protection Level              | 1                                |
     And waiting until the process ends
     Then my application contains the file "application.yml" with the content:
       """
       name: todo-app
       description: A todo application
       version: 0.0.1
-
       dependencies:
-        - name: exocom
-          version: 0.22.1
-
+      - name: exocom
+        version: 0.22.1
       services:
         public:
           html-server:
             location: ./html-server
-        private:
+        private: {}
```

Comment:
I don't think there is a way to get rid of `{}`. It could be pointer problems but I did try setting it to `nil`. I did manage to make it not initialize `config` map for the dependencies though.